docs/pages/cli-docs.mdx

```diff
@@ -532,22 +532,36 @@ registered devices using [`tsh mfa ls`](cli-docs.mdx#tsh-mfa-ls).
 
 ## tctl
 
-`tctl` is an admin CLI tool used to administer a Teleport cluster. It connects
-to an Auth Server, meaning that it must be run on the same host with the role
-`auth` . If you run `tctl` on a non- `auth` node it will show an error.
+`tctl` is an admin CLI tool used to administer a Teleport cluster. 
 
 `tctl` allows a cluster administrator to manage all resources in a cluster
 including nodes, users, tokens, and certificates.
 
 `tctl` can also be used to modify the dynamic configuration of the cluster, like
 creating new user roles or connecting trusted clusters.
 
+By default, `tctl` connects to a local Auth server, meaning it will fail if you 
+attempt to run `tctl` on a non-`auth` node.
+
+However it can also connect to a remote Auth server if an identity file and Auth/Proxy
+server IP address are provided. 
+
+<Admonition type="note">
+  Note that when a `tctl` command is run locally on an Auth server, the audit logs will 
+  show that it was performed by the Auth server itself. To properly audit admin actions
+  at scale, it is important to close off direct access to the Auth server and ensure
+  that admins connect remotely using identity files instead. 
+</Admonition>
+
 ### tctl Global Flags
 
 | Name | Default Value(s) | Allowed Value(s) | Description |
 | - | - | - | - |
 | `-d, --debug` | none | none | Enable verbose logging to stderr |
 | `-c, --config` | `/etc/teleport.yaml` | **string** filepath | Path to a configuration file |
+| `--auth-server` | none | **string** IP | `tctl` attempts to connect to a specific auth/proxy address instead of local auth |
```

Comment:
Does the auth server host have to be an IP address? Can it be a hostname, or even a `host:port` pair?